### DISCUSSION TITLE
Build for SKRME3V3 with Manual Mesh only

### DISCUSSION THREAD

**Discussion by leopinnheiro** [mriscoc/Ender3V2S1#730] [Ideas]
We already have a build for SKRME3V3 with BLTUBL, I think it would be interesting an MM-MPC build as well as for the other boards is it possible?
I currently have a SKRME3V3 but my BLTOUCH has gone down, a Manual Mesh only version would help a lot.

**Comment by mriscoc** [comment]
The configuration files for SKRM3E3 Manual Mesh version are in the special configuration repository
https://github.com/mriscoc/Special_Configurations#these-special-configurations-and-releases-are-sponsored-by-donors
